docs/references/kontena-yml.md

```diff
@@ -5,15 +5,60 @@ toc_order: 1
 
 # Kontena.yml reference
 
-Kontena.yml is a file in [YAML](http://yaml.org) format that defines a [Kontena application](../using-kontena/applications.md) with one or more [services]((../using-kontena/services.md)). It uses the same syntax and keys as a [Docker Compose file](https://docs.docker.com/compose/compose-file/). However, not all Docker Compose keys are supported in Kontena. The default name for this file is kontena.yml, although other filenames are supported.
+Kontena.yml is a file in [YAML](http://yaml.org) format that defines a [Kontena Stack](../using-kontena/stacks.md) with one or more [services]((../using-kontena/services.md)). It uses the same syntax and keys for services as a [Docker Compose file](https://docs.docker.com/compose/compose-file/). However, not all Docker Compose keys are supported in Kontena. The default name for this file is kontena.yml, although other filenames are supported.
 
-Each key defined in kontena.yml will create a service with that name prefixed with the project name. The image key is mandatory. Other keys are optional.
+Each service defined in kontena.yml will create a service with that name prefixed with the stack name. The image key is mandatory. Other keys are optional.
 
-You can use environment variables in configuration values with a bash-like ${VARIABLE} syntax. See [variable substitution](#variable-substitution) for full details.
+Each service within the same stack is both reachable by other services and discoverable by them at a hostname identical to the service name. See [networking](#networking) for full details.
 
+You can use environment variables in configuration values with a bash-like `${VARIABLE}` syntax. See [variable substitution](#variable-substitution) for full details.
+
+## Stack configuration reference
+
+#### stack
+Stack identification. Use format `<username>/<stack_name>`.
+```
+stack: kontena/jenkins
+```
+
+#### version
+Version number based on [Semantic Versioning](http://semver.org/).
```

Comment:
This must contain exactly three `major.minor.patch` version number components, in additional to the optional `-prelease` and `+buildmeta` suffixes.